tests/run.spec.js

```diff
@@ -30,4 +34,21 @@ it("Argument builds", async () => {
         const output = run.buildSaucectlArgs(input);
         expect(output).toStrictEqual(expected);
     }
+});
+
+it('Start saucectl', async () => {
+    childProcess.spawn.mockReturnValue({});
+    const tests = [{
+        returnValue: 0,
+        expectedValue: true
+    }, {
+        returnValue: 1,
+        expectedValue: false
+    }];
+    for (let i = 0; i < tests.length; i++) {
+        const testCase = tests[i];
+        helpers.awaitExecution.mockReturnValue(testCase.returnValue);
+        const status = await run.saucectlRun({});
+        expect(status).toBe(testCase.expectedValue);
+    }
```

Comment:
Maybe also add `expect(spawn.mock.calls).toMatchSnapshot()` (after the for-loop, not inside) so that we can validate  that the calls to "saucectl" are being tested.